pwa/app/routes/team.$teamNumber.($year).tsx

```diff
@@ -120,6 +124,32 @@ async function loadData(params: Params) {
     throw new Response(null, { status: 404 });
   }
 
+  // TODO: fetch these in parallel after initial render
```

Comment:
what is the proposed mechanism for doing this? ReactQuery?

Reply:
Yeah, most likely